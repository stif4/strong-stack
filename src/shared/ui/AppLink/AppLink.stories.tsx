import React from 'react';
import { Meta, StoryFn } from '@storybook/react';
import { AppLink, AppLinkTheme } from './AppLink';

export default {
    title: 'shared/AppLink',
    component: AppLink,
    args: {
        to: '/',
    },
} satisfies Meta<typeof AppLink>;

const Template: StoryFn<typeof AppLink> = (args) => <AppLink {...args} />;

export const Primary = Template.bind({});
Primary.args = {
    children: 'Text',
    theme: AppLinkTheme.PRIMARY,
};

export const Secondary = Template.bind({});
Secondary.args = {
    children: 'Text',
    theme: AppLinkTheme.SECONDARY,
};

export const Red = Template.bind({});
Red.args = {
    children: 'Text',
    theme: AppLinkTheme.RED,
};
