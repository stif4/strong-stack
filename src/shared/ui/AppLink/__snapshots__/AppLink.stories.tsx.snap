// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shared/AppLink Primary smoke-test 1`] = `
<div class="app dark">
  <a class="src-shared-ui-AppLink-AppLink-module__appLink src-shared-ui-AppLink-AppLink-module__primary"
     href="/"
     data-discover="true"
  >
    Text
  </a>
</div>
`;

exports[`shared/AppLink Red smoke-test 1`] = `
<div class="app dark">
  <a class="src-shared-ui-AppLink-AppLink-module__appLink src-shared-ui-AppLink-AppLink-module__red"
     href="/"
     data-discover="true"
  >
    Text
  </a>
</div>
`;

exports[`shared/AppLink Secondary smoke-test 1`] = `
<div class="app dark">
  <a class="src-shared-ui-AppLink-AppLink-module__appLink src-shared-ui-AppLink-AppLink-module__secondary"
     href="/"
     data-discover="true"
  >
    Text
  </a>
</div>
`;
